packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -92,6 +92,7 @@ abstract class ScrollView extends StatelessWidget {
     this.keyboardDismissBehavior = ScrollViewKeyboardDismissBehavior.manual,
     this.restorationId,
     this.clipBehavior = Clip.hardEdge,
+    this.isDefaultScrollAction = false,
```

Comment:
Semantically, how is this different from `primary` (line 84)?

Reply:
Oh that is probably a much better flag to use rather than introducing a new one. Thanks!